python-checks/src/main/java/org/sonar/python/checks/UnusedLocalVariableCheck.java

```diff
@@ -147,6 +143,21 @@ public PreciseIssue createIssue(SubscriptionContext ctx, Symbol symbol, Usage us
     }
   }
 
+  @Nullable
+  private static Symbol isUnderscoreSymbolAlreadyAssigned(SubscriptionContext ctx, Usage usage) {
```

Comment:
Since we only perform a null check on the return value, I'd suggest making that null check inside this method and returning a boolean instead.
That would also be more consistent with the method name.